homeassistant/components/androidtv/media_player.py

```diff
@@ -343,7 +342,7 @@ def __init__(
             self._attr_device_info[ATTR_MANUFACTURER] = manufacturer
         if sw_version := info.get(ATTR_SW_VERSION):
             self._attr_device_info[ATTR_SW_VERSION] = sw_version
-        if mac := format_mac(info.get(PROP_ETHMAC) or info.get(PROP_WIFIMAC, "")):
+        if mac := get_androidtv_mac(info):
             self._attr_device_info[ATTR_CONNECTIONS] = {(CONNECTION_NETWORK_MAC, mac)}
```

Comment:
```python
            self._attr_device_info[ATTR_CONNECTIONS] = {(CONNECTION_NETWORK_MAC, mac)}
```

Is this right? It's a set consisting of a single tuple?

Reply:
Yes, this is the format used to identify and correlate multiple devices with same MAC. I suppose it could have more tuple if there are multiple `ATTR_CONNECTIONS`